config/purposes/scrna_core_cdna_prep.wip.yml

```diff
@@ -26,7 +26,8 @@ LRC PBMC Pools:
 # This plate has come from faculty and is entering the SeqOps pipeline for the first time here.
 LRC PBMC Pools Input:
   :asset_type: plate
-  :stock_plate: false
+  # stock_plate has to be true so that it appears in the 'Purpose' list when generating a sample manifest in Sequencescape
+  :stock_plate: true
   :input_plate: false
```

Comment:
I was comparing this to Bioscan Lysate plate and it is similar there.

Reply:
Thanks for checking